### ISSUE TITLE
Drupal 7.60-7.61 commits: crossport or mark as NA

### ISSUE THREAD

**Issue by klonos** [backdrop/backdrop-issues#3375]
- [x] N/A | [d420d0a991](https://cgit.drupalcode.org/drupal/commit/?id=d420d0a991bda9cc621fcce8e16d173ba5aceaf9) | Back to 7.x-dev
- [x] N/A | [02012fbe94](https://cgit.drupalcode.org/drupal/commit/?id=02012fbe94e3df0f5730f694d717a135f7c792c8) | - Removing Larry Garfield from technical leadership positions.
- [x] https://github.com/backdrop/backdrop/pull/2363 | [a91bbe1937](https://cgit.drupalcode.org/drupal/commit/?id=a91bbe193723789831f295255c81cc317595971c) | [Issue #1815504 by pwolanin, Kevin Hankens, roderik, scor: file_save_upload() should not alter the file URI string before validation](http://drupal.org/node/1815504)
- [x] https://github.com/backdrop/backdrop/pull/2364 | [96753f64a2](https://cgit.drupalcode.org/drupal/commit/?id=96753f64a2e46f84c0cd27d4453720506f655e00) | [Issue #2091511 by Cameron Tod, mcdruid, mpdonadio, David_Rothstein, lokapujya, stefan.r, Berdir, alexpott, damiankloip, cosmicdreams, das-peter, heddn, xjm, catch, tstoeckler, anavarre, naveenvalecha, tim.plunkett, dawehner: Make cache_form expiration configurable, to mitigate runaway cache_form tables](http://drupal.org/node/2091511)
- [x] https://github.com/backdrop/backdrop/pull/2365 | [caf31116e9](https://cgit.drupalcode.org/drupal/commit/?id=caf31116e910a715806a2b1d00ed745103e48290) | [Issue #2873952 by tatarbj, augustofagioli, Dinesh18: Improve documentation regarding static JS or CSS in .info files](http://drupal.org/node/2873952)
- [x] https://github.com/backdrop/backdrop/pull/2366 | [09f11419b5](https://cgit.drupalcode.org/drupal/commit/?id=09f11419b52369b4fda1476fcd0403a56f1cba0d) | [Issue #2120459 by amitsedaiz, mahesh.gupta, Yago Elias, jhodgdon, joachim: theme_status_report() doesn't document its parameters](http://drupal.org/node/2120459)
- [x] https://github.com/backdrop/backdrop/pull/2367 | [b09c86eb17](https://cgit.drupalcode.org/drupal/commit/?id=b09c86eb170b6ae513c60a9d97a9487f7dc662d8) | [Issue #2555649 by mvc, mfb, rhclayto, Dinesh18: drupal_http_request() should allow caller to set the Host header](http://drupal.org/node/2555649)
- [x] https://github.com/backdrop/backdrop/pull/2368 | [44c38848b7](https://cgit.drupalcode.org/drupal/commit/?id=44c38848b794b5f4e53b1bee2671df59f42f889f) | [Issue #2899679 by Ayesh, Vj, MustangGB: PHP7 warning when uploading a user picture](http://drupal.org/node/2899679)
- [x] already implemented in #2494 | [a5eb776a37](https://cgit.drupalcode.org/drupal/commit/?id=a5eb776a3724261da2f4e64db531c0b402fc71cc) | [Issue #2790923 by stefanos.petrakis, jenlampton, lokapujya, alexpott, biguzis, zniki.ru, dimaro, benjf, themic8, subhojit777, thtas, kpv, Noe_, Berdir, thijsvdanker, Cottser, dcam, jhodgdon: Allow + symbol in usernames](http://drupal.org/node/2790923)
- [x] N/A | [01ff63a9cb](https://cgit.drupalcode.org/drupal/commit/?id=01ff63a9cb109533b006cadb2cc8ce15516d068e) | [Issue #2915900 by David_Rothstein, alexpott: MAINTAINERS.txt doesn't indicate that the database API is in need of a maintainer](http://drupal.org/node/2915900)
- [x] N/A - No need to specifically "Override" default image styles in Backdrop | [70aef6e5f6](https://cgit.drupalcode.org/drupal/commit/?id=70aef6e5f6e8c8cb56b456dc4e0e692259561372) | [Issue #2857976 by Darren Oh, Algarte: Remove undesired drag handle on first row of default image style effects](http://drupal.org/node/2857976)
- [x] N/A | [a63665d51c](https://cgit.drupalcode.org/drupal/commit/?id=a63665d51c2365dc671f0ba3c52d36e9692348ae) | [Issue #2857976: Remove undesired drag handle on first row of default image style effects (followup to add a code comment)](http://drupal.org/node/2857976)
- [x] https://github.com/backdrop/backdrop/pull/2369 | [4c7cb08a1e](https://cgit.drupalcode.org/drupal/commit/?id=4c7cb08a1eaa4f5ea09a3304b270bbcb34f69cb0) | [Issue #2889279 by David_Rothstein, Daniel_Rose: Fix documentation of taxonomy term title callback code to match nodes and to not imply that it sets the title of the term page](http://drupal.org/node/2889279)
- [x] https://github.com/backdrop/backdrop/pull/2370 | [0a1851cfd5](https://cgit.drupalcode.org/drupal/commit/?id=0a1851cfd542ba8b968bec38b72524f28fc87a87) | [Issue #2857606 by selvanathan, mithenks, racordero94, PunkAntifa: Fix incorrect documentation link at admin/config/media/file-system](http://drupal.org/node/2857606)
- [x] https://github.com/backdrop/backdrop/pull/2371 | [6336b7177e](https://cgit.drupalcode.org/drupal/commit/?id=6336b7177eaf1b93bc8694056b53ab8f22c274f5) | [Issue #2918570 by xjm, David_Rothstein: Hotfix for ImageToolkitGdTestCase failing on newer PHP 7 and 7.1 releases](http://drupal.org/node/2918570)
- [x] N/A | [dcf8c646fb](https://cgit.drupalcode.org/drupal/commit/?id=dcf8c646fbab4f9fbb0d1c2cf85b2bcfdceaf433) | Fix CHANGELOG.txt entry for Drupal 7.58 to match the corresponding security advisory.
- [x] already implemented in #2948 | [127ffe91ff](https://cgit.drupalcode.org/drupal/commit/?id=127ffe91fffb5813c32490672d68c824595d75a0) | [Issue #2946045 by nicorac, sjerdo, soyjaz, aprogs, Göran, joseph.olstad, Fabianx: Unable to update modules due to Archive_Tar incompatibility with PHP 7.2](http://drupal.org/node/2946045)
- [x] Implemented in #2948| [28de677281](https://cgit.drupalcode.org/drupal/commit/?id=28de6772813387bf02a4bf6f75b9119c4c20a3f4) | [Issue #2925449 by bkosborne, oriol_e9g, Ayesh, sjerdo, mikeytown2, piggito, vprocessor, JayKandari, delta, joseph.olstad, andypost, Fabianx, David_Rothstein, Neo13, yogeshmpawar, amme: [PHP 7] Function each() is deprecated since PHP 7.2 [D7]](http://drupal.org/node/2925449)
- [x] https://github.com/backdrop/backdrop/pull/2372 | [73e12f0ddf](https://cgit.drupalcode.org/drupal/commit/?id=73e12f0ddf1ed60c13331dc3a9b8ea16d1599307) | [Issue #2885610 by joseph.olstad, Ayesh, sjerdo, Fabianx, JamesOakley, David_Rothstein, Bahson: [PHP 7.2] Avoid count() calls on uncountable variables](http://drupal.org/node/2885610)
- [x] N/A | [76a4dc7098](https://cgit.drupalcode.org/drupal/commit/?id=76a4dc7098771a14e7d64a827457dcadd7feafbc) | [Issue #2885129 by Ayesh, bkosborne, kiamlaluno, Fabianx: [PHP 7.2] create_function() is deprecated](http://drupal.org/node/2885129)
- [x] https://github.com/backdrop/backdrop/pull/2373 | [b014c196e1](https://cgit.drupalcode.org/drupal/commit/?id=b014c196e1eab417e0703c3fc77e340a213a8824) | [Issue #2900373 by David_Rothstein, joseph.olstad, jollysolutions, joelpittet: [D7] When a theme has a managed_file field and a submit callback, call to undefined function error is thrown](http://drupal.org/node/2900373)
- [x] https://github.com/backdrop/backdrop/pull/2374 | [083a4eca4a](https://cgit.drupalcode.org/drupal/commit/?id=083a4eca4a2ebc5276eed2f11fc4a595b07898ba) | [Issue #2996519 by DamienMcKenna, joseph.olstad, Pol: PHP 7.2 compatibility - make sure some variables are arrays in theme_links()](http://drupal.org/node/2996519)
- [x] N/A | [270588129c](https://cgit.drupalcode.org/drupal/commit/?id=270588129ccb8514ff64a91ef29a47bee0f4f6e2) | Back to dev.
- [x] https://github.com/backdrop/backdrop/pull/2375 | [36d7b050d7](https://cgit.drupalcode.org/drupal/commit/?id=36d7b050d78a49150587da3ac06020ebd71b85b3) | [Issue #2843240 by droplet, donquixote, joseph.olstad, hanoii, mw4ll4c3, dsnopek, davidwatson, idebr, mforbes, Pol, bwaindwain, svenryen, nattie: IE11 & Chrome(PointerEvents enabled) & some Firefox scroll to the top of the page after dragging the bottom item with jquery 1.5 <-> 1.11](http://drupal.org/node/2843240)
- [x] N/A | [bfefcae224](https://cgit.drupalcode.org/drupal/commit/?id=bfefcae224dd63b62395e373c491b5630ebf21af) | [Issue #2982027: Add Pol as provisional Drupal 7 branch maintainer](http://drupal.org/node/2982027)
- [x] N/A | [b446a96396](https://cgit.drupalcode.org/drupal/commit/?id=b446a9639643ba1331cbf8da55782fa16f4c50f9) | Update changelog
- [x] basically the same as https://github.com/backdrop/backdrop/pull/2372 - just fixes a typo | [c7ce87b281](https://cgit.drupalcode.org/drupal/commit/?id=c7ce87b281b42fd803d3c7ccb924cc0f741580ef) | [Issue #2885610 by David_Rothstein: Follow-up to [PHP 7.2] Avoid count() calls on uncountable variables](http://drupal.org/node/2885610)
- [x] included in https://github.com/backdrop/backdrop/pull/2372 | [1664b8b69b](https://cgit.drupalcode.org/drupal/commit/?id=1664b8b69bbc144e7d9439091e92e4e599a2aab2) | [Issue #2988184 by jacob.embree, Pol, sjerdo, Jorrit: Fix call to count() on non-countable $links in theme_links()](http://drupal.org/node/2988184)
- [x] N/A | [55d17c3e4d](https://cgit.drupalcode.org/drupal/commit/?id=55d17c3e4dcc888984795c7e7e6a9d745708e275) | Drupal 7.61

git command
```
git log 7.60..7.61 --reverse --no-merges --pretty=format:"- [ ] #xx | [%h](https://cgit.drupalcode.org/drupal/commit/?id=%H) | [%s](http://drupal.org/node/)"
```
REGEX: Find: `(Issue #)(\w\w+)(.*?)(node\/)` Replace: `$1$2$3$4$2`
Find: `http://drupalcode.org/project/drupal.git/commit/` Replace: `https://cgit.drupalcode.org/drupal/commit/?id=`

**Comment by klonos** [comment]
OK, PRs up for review. Most of them are pretty straight-forward ports. For those that I was unsure, I have added `???`.

In general, all could use a second pair of eyes.

**Comment by herbdool** [comment]
@klonos since they're all on the same issue what's the best way to mark each as RTBC? Check it off or put a note on that line?

**Comment by herbdool** [comment]
I don't see an approval button on the PRs but perhaps it's because I'm using mobile right now. So check them off here and we update the issue label when most of them are ready?

**Comment by klonos** [comment]
@herbdool you need to go to the "Files changed" tab in the PR, where you will find a "Review changes" drop-button in the top-right:

![screen shot 2018-11-27 at 8 18 38 am](https://user-images.githubusercontent.com/2423362/49042781-31e71000-f21d-11e8-93f6-dd099ee4ee7d.png)

...add a comment (required), then finally click "Approve" 😉 

But as said, ticking the respective boxes in the issue summary works the same.

**Comment by quicksketch** [comment]
A lot of these are RTBC, so I'm bumping this up that status. Once we commit all of those, we can move this back to needs review (or whatever the right status is) for remaining issues.

**Comment by jenlampton** [comment]
Looks like all are RTBC except these two:

* https://github.com/backdrop/backdrop/pull/2375 - needs @quicksketch feedback
* https://github.com/backdrop/backdrop/pull/2371 - needs review

and 

> mostly implemented in #2948 ??? - need to check the changes that went in locale.test

**Comment by quicksketch** [comment]
I've merged or confirmed issues as not necessary in Backdrop for all remaining items. Great job @klonos on these ports and tons of thanks to @herbdool for reviewing them and adding your fixes!

The only issue that this seems to bring up for me is that `Issue #2925449 by bkosborne, oriol_e9g, Ayesh, sjerdo, mikeytown2, piggito, vprocessor, JayKandari, delta, joseph.olstad, andypost, Fabianx, David_Rothstein, Neo13, yogeshmpawar, amme: [PHP 7] Function each() is deprecated since PHP 7.2 [D7]` was already fixed in Backdrop in https://github.com/backdrop/backdrop-issues/issues/2948, but I did a search on the entire codebase for `each()` calls and we have 4 remaining calls that were not fixed in the previous patch. Those 4 places are all from contrib though, which explains why we didn't see them in D7 core. There is an existing issue to fix these at https://github.com/backdrop/backdrop-issues/issues/3403.
